src/Program.ts

```diff
@@ -1038,6 +1035,11 @@ export class Program {
             return this.files[
                 (normalizePath ? util.standardizePath(filePath) : filePath).toLowerCase()
             ] as T;
+        } else if (util.isUriLike(filePath)) {
```

Comment:
I'm wondering if we could eliminate this separate condition, in favor of doing something like:

```typescript
else {
           filePath = util.uriToPath(filePath);
            return this.destMap.get(
                (normalizePath ? util.standardizePath(filePath) : filePath).toLowerCase()
            ) as T;
        }

Reply:
What if a file's destPath is different from its srcPath and the lookup is the uri of the srcPath?

Reply:
yeah, good point. I think I looked at the wrong condition. I think it technically should merge with the condition ABOVE this one, not the `else` below. But now that regex is getting a bit busy. Yeah, just leave it as is. we can fix it later if there are performance issues.